specs/WinRT/WinRTAPIContracts.md

```diff
@@ -22,18 +22,22 @@ WinRT APIs.
    1. One contract per feature/component/sub-system
```

Comment:
Should add a note that [ApiInformation.IsApiContractPresent](https://docs.microsoft.com/uwp/api/Windows.Foundation.Metadata.ApiInformation.IsApiContractPresent) doesn't currently work against WinAppSDK contracts.

Reply:
Because https://task.ms/36717508 or something else?

Reply:
That's a surprise; IsApiContractPresent should be using the `Ro...` APIs that everything else uses.

Reply:
[Predefined attributes (MIDL 3.0) - Windows UWP applications | Microsoft Docs](https://docs.microsoft.com/en-us/uwp/midl-3/predefined-attributes#the-contract-attribute)
![image](https://user-images.githubusercontent.com/72222/160949464-1a1a3102-8a46-4c74-b6aa-2d14551e4ef1.png)